parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -361,6 +378,7 @@ impl Default for ProxyType {
 		Self::Any
 	}
 }
+// TODO:check-parameter - setup ProxyType for ForeignAssets?
```

Comment:
I was thinking no (based on [this comment](https://github.com/paritytech/cumulus/pull/2167/files#diff-67f8d30fd426048ab88d224694ba10c5f225eb66b5afdcdcb10ead450e6d2c02R277)), but am also not strongly opposed to it. It's more of a guess about how people might use this, but perhaps they would want proxies.

Reply:
I removed that TODO, so maybe lets wait for demands from comunity